src/main/java/com/microsoft/aad/adal4j/AuthenticationContext.java

```diff
@@ -24,40 +24,21 @@
 package com.microsoft.aad.adal4j;
 
 import javax.net.ssl.SSLSocketFactory;
-import java.io.UnsupportedEncodingException;
-import java.net.MalformedURLException;
-import java.net.Proxy;
-import java.net.URI;
-import java.net.URL;
-import java.net.URLEncoder;
-import java.security.MessageDigest;
-import java.security.NoSuchAlgorithmException;
+import java.net.*;

```

Comment:
>java.net.*; [](start = 7, length = 11)

is this needed. Why not being specific? Does the compiler optimize not needed libs out?